### PR TITLE
Update URLs, bump build number

### PR DESCRIPTION
**Pull request by dsherry**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by dsherry** [comment]
@conda-forge-admin, please rerender

**Comment by freddyaboulton** [comment]
@dsherry - not sure what happened with the github action but I re-rendered locally and pushed changes. This should be good to merge once CI passes!

**Review by freddyaboulton — APPROVED** [review]

**Comment by dsherry** [comment]
Thanks @freddyaboulton ! Yeah, I guess there's an "Allow edits from maintainers" checkbox I was supposed to check when I created the PR. [Here's the github doc on how to do that](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork). Next time!

**Pull request merged**
